### ISSUE TITLE
[API 수정 요청] 이사 요청 목록 조회(by-mover) 응답 수정

### ISSUE THREAD

**Issue by ajantang** [codeit-moving/moving-be#83]
### 응답 수정
isQuoted, isPastRequest가 적용된 응답 정보 필요

- 적용이 필요한 정보
  - serviceCounts
  - requestCounts

아래 스크린 샷을 보시면 아시겠지만, 필터링에 표시를 위한 Counts 정보와 리스트 정보가 일치하지 않습니다

### 스크린샷
![image](https://github.com/user-attachments/assets/ff6692a8-25ad-4a82-a9b7-2aba0a982e50)
![image](https://github.com/user-attachments/assets/5a13b72a-296a-4dc9-b858-3603fe47bb88)
![image](https://github.com/user-attachments/assets/4b7d5393-8b44-44a0-b533-e0a3e048fc89)

**Comment by WooHyunTak** [comment]
![image](https://github.com/user-attachments/assets/42a08dda-6c13-46f2-b918-d5a6695aa0eb)

조회되는 counts에 isQuoted, isPastRequest 필터링은 하고 service, isDesignated, keyword 는 필터링을 안 한 counts 값으로 반환해야 한다고 이해하면 될 까요??

**Comment by ajantang** [comment]
네. 결과적으로 조회되는 결과(리스트)의 카테고리별 분포를 확인할 수 있는 Counts 정보와 실제 조회되는 결과가 불일치하게 출력되는 것이 문제라. 이 부분을 일치하게 해줘야 합니다.

**Comment by WooHyunTak** [comment]
![image](https://github.com/user-attachments/assets/94f395ae-09a8-42f3-9e1f-1e43e3f3ad1a)
이 결과에서는 
serviceCounts에 스몰 : 2 나머지는 0
requestCounts에는 총 이사요청수와 총 지정요청수 가 나오면 되는 거죠?

**Comment by ajantang** [comment]
정확히 말씀드리자면, 

Counts 정보 조회 조건(쿼리)
- smallMove=true // 소형이사 요청 포함
- houseMove=true // 가정이사 요청 포함
- officeMove=true // 사무실이사 요청 포함
- isQuoted=false // 확정/반려된 요청 제외
- isPastRequest=false // 이사일 지난 요청 제외

조건이어야 합니다. 그래야 기사의 입장에서 필터링을 통해 유효한 요청 목록을 확인 가능하며, 최신 요청 수를 UI에서 보여줄 수 있습니다.

위에 코멘트로 남겨주신 response 예시의 경우, 앞서 언급한 Counts 정보 조회 조건에 부합한 결과라면 상관없지만, houseMove=false 라던가, isDesignated=false 와 같은 다른 필터링 조건에 영향을 받은거라면, 원하는 결과가 아닙니다.

**Comment by ajantang** [comment]
수정요청 반영 확인했습니다
